lib/dart_sass.ex

```diff
@@ -183,21 +169,7 @@ defmodule DartSass do
       stderr_to_stdout: true
     ]
 
-    args = config_args ++ extra_args
-    path = bin_path()
-
-    # TODO: Remove when dart-sass will exit when stdin is closed.
-    # Link: https://github.com/sass/dart-sass/pull/1411
-    path =
-      if "--watch" in args and platform() != :windows do
-        [script_path() | path]
-      else
-        path
-      end
-
-    path
-    |> cmd(args, opts)
-    |> elem(1)
+    run_cmd(bin_paths(), config_args ++ extra_args, opts) |> elem(1)
```

Comment:
We still need `script_path()` unfortunately. :( The upstream PR was merged but the changes had to be reverted due to issues on Windows.

Reply:
Ah, that's a pity, but okay. Done!